### PR TITLE
feat: force network adapter

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: vc-worker.py [hunk 1]
```diff
@@ -50,10 +50,9 @@ def process_deploy_action(conn, action, vc):
 
         template = get_template(machine_ro.labels)
 
-        if Settings.app['vsphere']['default_network_name']:
+        network_interface = get_network_interface(machine_ro.labels)
```
Comment (petrSchreiber) [inline vc-worker.py:53]: I would suggest to move this to else branch for the if below.

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
Minor stylistic change, otherwise I perceive this as improvement compared to the previous state.

File: vc-worker.py (hunk 1 above)
Comment (tomas-danek) [reply vc-worker.py:53]: +1 from me, it will be more efficient and change even more readable :)

**Review by tomas-danek — COMMENTED** [review]
I'd vote for move getting the  network interface to `else` as it was before.

File: vc-worker.py [hunk 2]
```diff
@@ -50,9 +50,9 @@ def process_deploy_action(conn, action, vc):
 
         template = get_template(machine_ro.labels)
 
-        if Settings.app['vsphere']['default_network_name']:
+        if Settings.app['vsphere']['default_network_name'] and Settings.app['vsphere']['force_default_network_name']:
             network_interface = Settings.app['vsphere']['default_network_name']
-        else:
+        else
```
Comment (tomas-danek) [inline vc-worker.py:55]: there's `:` for the `else` clause

**Review by tomas-danek — CHANGES_REQUESTED** [review]
`:` missing

**Review by tomas-danek — APPROVED** [review]
nice and readable 👍

**Review by petrSchreiber — APPROVED** [review]
🎖️ Thank you for the changes

**Pull request merged**